modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -149,42 +149,13 @@ properties:
               `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
 
               Starting Sync Gateway with `import_docs` enabled will generate the mobile-specific metadata for all the existing documents in the Couchbase Server bucket. With shared_bucked_access enabled, documents can be mutated on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API and these changes will be seamlessly synchronized over to the connected Couchbase Lite enabled clients.
-
```

Comment:
Addresses @rajagp comment to move this to shared-bucket-access.html (https://github.com/couchbase/docs-sync-gateway/pull/111#discussion_r350274783)